library/src/main/java/com/firebase/ui/auth/core/FirebaseLoginDialog.java

```diff
@@ -28,20 +25,24 @@
     TwitterAuthHelper mTwitterAuthHelper;
     GoogleAuthHelper mGoogleAuthHelper;
     PasswordAuthHelper mPasswordAuthHelper;
-
     TokenAuthHandler mHandler;
     Firebase mRef;
     Context mContext;
-    public Boolean isActive = false;
+    View mView;
 
     public void onStart() {
         super.onStart();
-        isActive = true;
+        if (mGoogleAuthHelper != null) mGoogleAuthHelper.onStart();
```

Comment:
Why is this only needed for the GoogleAuthProvider and not for the others? It might be worth mentioning that reason in a comment.